components/scream/tests/coupled/dynamics_physics/homme_shoc_cld_p3_rrtmgp/input.yaml

```diff
@@ -20,7 +20,7 @@ Atmosphere Processes:
   Schedule Type: Sequential
   Process 0:
     Process Name: Homme
-    Enable Output Field Checks: false
+    Enable Pre Run Property Checks: false
```

Comment:
Now that I think of it: should we call these `Precondition Property Checks`? And `Postcondition Property Checks`?